test/integration/util_command.go

```diff
@@ -153,7 +153,7 @@ func executeCmdWithTimeOutNilAssertion(
 func executeWithTimeout(cmd *cobra.Command, timeout time.Duration, args ...string) error {
 
 	args = append(args, flag(params.DebugFlag), flag(params.RetryFlag), "3", flag(params.RetryDelayFlag), "5")
-	args = appendProxyArgs(args)
```

Comment:
@igorlombacx  please remove this, it should only be disabled in the local environment.

Reply:
done :)